src/application/modules/auth/useCases/logout/LogoutUseCase.test.ts

```diff
@@ -1,22 +1,20 @@
-import { ILogProvider } from "../../../../shared/log/providerContracts/ILogProvider";
-import { ApplicationErrorMock } from "../../../../mocks/ApplicationError.mock";
 import applicationStatus from "../../../../shared/status/applicationStatus";
-import { LocaleTypeEnum } from "../../../../shared/locals/LocaleType.enum";
-import { UseCaseTraceMock } from "../../../../mocks/UseCaseTrace.mock";
-import { IAuthProvider } from "../../providerContracts/IAuth.provider";
-import { ISession } from "../../../../../domain/session/ISession";
-import { SessionMock } from "../../../../mocks/Session.mock";
 import appMessages from "../../../../shared/locals/messages";
```

Comment:
Please organize imports from longest to shortest like before your changes.
I know, but code is art in some way, so we have to try to make it perfect.

Reply:
It was a failure to understand the design pattern on my part. LOL
Correcting here